### ISSUE TITLE
Feature: Colab playground

### ISSUE THREAD

**Issue by mkh-welsch** [reductstore/reduct-py#46]
### Current problem

Testing reduct storage ist hard right now, because it is not very compatible so far and one as to figure out the best environment at first and set it up.

### Possible solution

To test reduct storage, if it fits once need for a AI engineer, it would be nice to have a ready to go playground.  Colab would be very helpful as a reference implementation.
[reduct_storage_test.ipynb.zip](https://github.com/reduct-storage/reduct-py/files/9846346/reduct_storage_test.ipynb.zip)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by atimin** [comment]
Relates to #45

**Comment by atimin** [comment]
@mkh-welsch we released v1.1.0 with support of Python 3.7. I've tried to install it on Google Colabs, this works:

```
!pip install reduct-py>=1.1.0
```

I didn't catch how to access to the  fast API endpoint , when it runs on Colabs but you script should work. Only one change is needed:

```python
    client = Client('https://play.reduct-storage.dev', api_token="reduct")
```

The playground uses token authorization, so you need use `reduct`as a full access token. 

Fill free to ask if you have any questions. And if it is working on your side then, please, close the issue.
